apps/native/components/ui/ScanQRCodeCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { StyleSheet, Text, View, TouchableOpacity, Image } from "react-native";
+import { THEME, TypographyWeight } from "@/theme";
+
+export default function ScanQRCodeCard({ onPress }: { onPress?: () => void }) {
+  const { colors, spacing, typography, layout } = THEME;
+
+  return (
+    <TouchableOpacity
```

Comment:
TouchableOpacity is legacy API. 
while it still works, it's no longer the recommended choice in the modern React Native. 
Pressable provides better control over interaction state and is more future proof. 

The Accessibility is missing.
TouchableOpacity here does not define accessibilityRole, accessibilityLabel, or accessibilityHint, which makes this action unclear to screen readers.